pages/processetti/metodologia/index.tsx

```diff
@@ -1,16 +1,194 @@
+          persistenti (come ad esempio le persone o i luoghi geografici, ma
+          anche classi più astratte, come le professioni), e le{" "}
+          <i>temporal entities</i> (o <i>events</i>), che includono fenomeni
+          situati nel tempo in grado di caratterizzare le istanze delle classi
+          persistenti (per esempio la nascita di un individuo, la sua
+          occupazione o la sua localizzazione in un dato momento).
+          Contrariamente ai dati strutturati in un modello orientato agli
+          oggetti (<i>object-oriented structured data model</i>), quello
+          event-centered, oltre ad assicurare una ricca discretizzazione,
+          garantisce una maggiore flessibilità nel trattare le incertezze e i
+          cambiamenti nel tempo degli &quote;attributi&quote; delle entità
+          persistenti (<a href="#2022">Beretta, 2022</a>).
+        </p>
+        <p>
+          <ImageWrapper
+            legend={"Event centered data model"}
+            imageUrls={[""]}
+            dialogChildren={
+              <Image
+                src={img2}
+                alt="Schematizzazione del Event centered data model tratta dalla documentazione di KleioLab"
+              />
+            }
+          >
+            <Image
+              src={img2}
+              placeholder="blur"
+              width={608}
+              height={280}
+              objectFit={"scale-down"}
+              alt="Schematizzazione del Event centered data model tratta dalla documentazione di KleioLab"
+            />
+          </ImageWrapper>
+        </p>
+        <p>
+          A seguito di diversi cicli di <i>information production</i> -
+          comprensivi della trascrizione dei processetti, della creazione di
+          dati strutturati e dell’annotazione semantica delle trascrizioni, si è
+          dato corso alla progressiva puntualizzazione del modello, nell’ottica
+          di renderlo il più possibile rispondente ai quesiti di ricerca. Ciò ha
+          portato, all’individuazione e conseguente attivazione di numerose
+          categorie concettuali (classi e proprietà) per la descrizione delle
+          entità persistenti e degli <i>event</i>, nonché alla definizione di
+          numerosi vocabolari controllati, capaci di dare forma a un ampio
+          spettro di informazioni relative agli attori coinvolti, al ruolo di
+          ciascuno nell’ambito dell’inchiesta, alle loro relazioni sociali e
+          alla mobilità <i>intra</i> ed <i>extra</i> urbana, non meno che alla
+          natura dei singoli processetti.
+        </p>
+        <br />
+        <br />
+        <br />
+
+        <p>
+          Per una più analitica descrizione della metodologia e della semantica
+          sottesa alla definizione infrastrutturale di Geovistory si vedano:
+        </p>
+        <p>
+        <a name="2013">
+          Beretta Francesco, Butez Charlotte (2013), “Un SIG Collaboratif Pour
```

Comment:
The `<a>`-tag is the link you click on. The `href=#2013` will target the element with `id=2013` on that page. I suggest this: 

```suggestion
        <p id="2013">
          Beretta Francesco, Butez Charlotte (2013), “Un SIG Collaboratif Pour
          La Recherche Historique”, in “Géomatique Expert”, 91, pp. 30-35 / 92,
          pp. 48-54
        </p>

        <p id="2016">
          Beretta Francesco (2016), “Pour une annotation sémantique des textes:
          le projet symogih.org et la Text encoding initiative”, in “Bruniana e
          Campanelliana, Ricerche filosofiche e materiali storico - testuali”,
          XXII.2 , pp. 453-65 [https://doi.org/10.19272/201604102005]
        </p>

        <p id="2021">
          Beretta Francesco (2021), “A Challenge for Historical Research: Making
          Data FAIR Using a Collaborative Ontology Management Environment
          (OntoME)”, in “Semantic Web”, 12.2, pp. 279-94
          [https://doi.org/10.3233/SW-200416]
        </p>

        <p id="2022">
          Beretta Francesco (2022), “Interopérabilité Des Données de La
          Recherche et Ontologies Fondationnelles: Un Écosystème d”extensions Du
          CIDOC CRM Pour Les Sciences Humaines et Sociales”, in N. Lasolle, O.
          Bruneau, J. Lieber (a cura di),{' '}
          <i>Actes Des Journées Humanités Numériques et Web Sémantique</i>,
          Nancy, pp. 2-22 [https://doi.org/10.5281/zenodo.7014341]
        </p>

        <p id="2023">
          Beretta Francesco (2023), “Données ouvertes liées et recherche
          historique: un changement de paradigme”, in “Humanités numériques”, 7
          [https://doi.org/10.4000/revuehn.3349]
        </p>
```